js/ui/tooltips/number-scrubber.js

```diff
@@ -79,6 +79,14 @@ TooltipEngine.classes.numberScrubber = TooltipBase.extend({
                     self.$el.addClass("dragging");
                     $(this).css("visibility", "hidden");
                     self.trigger("scrubbingStarted");
+                    // The text-to-be-tweaked needs to be the same length at the start and end
+                    // of the anti-undo changes.
+                    // I could probably just remember the length, but I like putting back the
+                    // original string. (It might even matter for i18n.)
+                    var Range = ace.require("ace/range").Range;
+                    var loc = self.aceLocation;
+                    var range = new Range(loc.row, loc.start, loc.row, loc.start + loc.length);
+                    self.originalString = parent.editor.getTextRange(range);
```

Comment:
TBH I'd been trying to resolve the errors as they came up - I tried those commands and now I'm getting a different error. I ran the suggested git commands and still got the same error. (I'm running on Windows.)

F:\khan\live-editor-fork>gulp
[10:24:13] Using gulpfile F:\khan\live-editor-fork\gulpfile.js
[10:24:13] Starting 'watch'...
[10:24:13] Finished 'watch' after 628 ms
[10:24:13] Starting 'check'...
missing files:
  external/iframe-overlay/dist/iframe-overlay.js
  external/iframe-overlay/dist/iframe-overlay.js
make sure to run:
  git submodule sync && git submodule update
Aborting build

F:\khan\live-editor-fork>